src/cclib/parser/data.py

```diff
@@ -382,7 +382,7 @@ def nelectrons(self):
 
     @property
     def closed_shell(self):
-        return orbitals.orbitals(self).closed_shell()
+        return orbitals.Orbitals(self).closed_shell()

```

Comment:
What's with `orbitals` being changed to `Orbitals`?

Reply:
`orbitals` was mostly a typo, since the name of the class is `Orbitals` so I changed it accordingly. If you try to run closed_shell with `orbitals.orbitals`, it gives the following error:
`AttributeError: 'module' object has no attribute 'orbitals'`

Reply:
Ok.